libr/arch/p/x86/plugin_cs.c

```diff
@@ -32,10 +37,51 @@ call = 4
 
 #define CSINC X86
 #define CSINC_MODE \
-	(a->config->bits == 64)? CS_MODE_64: \
-	(a->config->bits == 32)? CS_MODE_32: \
-	(a->config->bits == 16)? CS_MODE_16: 0
-#include "capstone.inc.c"
+	(as->config->bits == 64)? CS_MODE_64: \
+	(as->config->bits == 32)? CS_MODE_32: \
+	(as->config->bits == 16)? CS_MODE_16: 0
+#include "../capstone.inc.c"
+typedef struct plugin_data_t {
+	CapstonePluginData cpd;
+	RRegItem reg; // ??
```

Comment:
```suggestion
	RRegItem reg; // XXX ??
```
(So we don't forget)